eng/common/cross/toolchain.cmake

```diff
@@ -48,11 +48,13 @@ elseif(TARGET_ARCH_NAME STREQUAL "arm64")
   set(CMAKE_SYSTEM_PROCESSOR aarch64)
   if(EXISTS ${CROSS_ROOTFS}/usr/lib/gcc/aarch64-alpine-linux-musl)
     set(TOOLCHAIN "aarch64-alpine-linux-musl")
-  else()
+  elseif(LINUX)
     set(TOOLCHAIN "aarch64-linux-gnu")
-  endif()
-  if(TIZEN)
-    set(TIZEN_TOOLCHAIN "aarch64-tizen-linux-gnu/9.2.0")
+    if(TIZEN)
+      set(TIZEN_TOOLCHAIN "aarch64-tizen-linux-gnu/9.2.0")
+    endif()
+  elseif(FREEBSD)
+    set(triple "aarch64-unknown-freebsd12")
   endif()
```

Comment:
should we keep the "else" or is the LINUX covering all known cases?

Reply:
This is a mirror of https://github.com/dotnet/arcade/pull/9834. It is better if we don't set the toolchain unconditionally to linux CMake has broader platform and toolchain coverage, so we should explicitly specify the configuration for each supported platform when doing the port work.